test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -6523,8 +6523,13 @@ def manager(self, cmd, obj_type, attrib=None, id=None, extend=None,
                                 op = '='
                             # handle string arrays as double quotes if
                             # not already set:
-                            if isinstance(v, str) and ',' in v and v[0] != '"':
-                                v = '"' + v + '"'
+                            if isinstance(v, str):
+                                if ',' in v and v[0] != '"':
+                                    v = '"' + v + '"'
+                                elif any((c in v) for c in set(", \n'")):
+                                    v = '"%s"' % v
+                                elif '"' in v:
+                                    v = "'%s'" % v
```

Comment:
What if the string has a double quote " and a space?  Wouldn't there be a problem?